main.py

```diff
@@ -2,6 +2,38 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres_list = ["Western", "Action", "Dramma"]
+    [Genre.objects.create(name=item, ) for item in genres_list]
+    actors_list = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    [Actor.objects.create(first_name=actor[0], last_name=actor[1])
+     for actor in actors_list]
+    Genre.objects.filter(
+        name="Dramma"
+    ).update(name="Drama")
+    Actor.objects.filter(
+        last_name="Klooney"
+    ).update(last_name="Clooney")
+    Actor.objects.filter(
+        first_name="Kianu"
+    ).update(first_name="Keanu", last_name="Reeves")
+    Genre.objects.filter(
+        name="Action"
+    ).delete()
+    Actor.objects.filter(
+        first_name="Scarlett"
+    ).delete()
+    smith_actors = Actor.objects.filter(
```

Comment:
There is no need to allocate `smith_actors` variable, return the queryset directly.